src/databricks/labs/ucx/source_code/linters/table_creation.py

```diff
@@ -115,6 +113,7 @@ def lint(self, code: str) -> Iterable[Advice]:
         if self._skip_dbr:
             return
 
-        tree = ast.parse(code)
-        for node in ast.walk(tree):
-            yield from self._linter.lint(node)
+        linter = ASTLinter.parse(code)
+        for node in TreeWalker.walk(linter.root):
+            advices = list(self._linter.lint(node))
+            yield from advices
```

Comment:
I'm curious about why we materialize the collection here? (Leftover from debugging?)